File: typescript-angular/API/Client/ModifiedQueue.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface ModifiedQueue {

        "aCL"?: ACL;

        "type"?: string;

        "expires"?: Date;

        "queueType"?: string;

        "retry"?: string;

        "name"?: string;

        "subscribers"?: Array<string>;

        "isModified"?: boolean;

        "modifiedColumns"?: Array<string>;

        "messages"?: Array<QueueMessage>;
    }

}

File: akka-scala/src/main/scala/io/swagger/client/model/ModifiedQueue.scala
package io.swagger.client.model

import io.swagger.client.core.ApiModel
import org.joda.time.DateTime


case class ModifiedQueue (
  aCL: Option[ACL],
  `type`: Option[String],
  expires: Option[DateTime],
  queueType: Option[String],
  retry: Option[String],
  name: Option[String],
  subscribers: Option[Seq[String]],
  isModified: Option[Boolean],
  modifiedColumns: Option[Seq[String]],
  messages: Option[Seq[QueueMessage]])
   extends ApiModel


